### PR TITLE
Add tests to catch invalid input edge cases

### PR DESCRIPTION
**Pull request by McEileen**
I added version number 2.0.0 to track the changes in tests.  Let me know if there are any questions!

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/xjava/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

**Comment by McEileen** [comment]
Agh, the travis build failed.  I will look into why it failed and make fixes.

**Comment by stkent** [comment]
It looks like missing semicolons is causing compilation to fail :)

File: exercises/rna-transcription/src/test/java/RnaTranscriptionTest.java [hunk 1]
```diff
@@ -5,6 +5,13 @@
 
 public class RnaTranscriptionTest {
 
+    /*
+    version: 2.0.0
+     */
+
+    @Rule
```
Comment (Smarticles101) [inline exercises/rna-transcription/src/test/java/RnaTranscriptionTest.java]: Rule wasn't imported :)

File: exercises/rna-transcription/src/test/java/RnaTranscriptionTest.java [hunk 2]
```diff
@@ -5,6 +5,13 @@
...
+    public ExpectedException thrown = ExpectedException.none();
```
Comment (Smarticles101) [inline exercises/rna-transcription/src/test/java/RnaTranscriptionTest.java]: ExpectedException wasn't imported either :)

**Comment by McEileen** [comment]
Thanks for the pointers!  Intellij wasn't doing red underwave marks for my compilation errors, like the missing semicolons, and it also didn't highlight `ExpectedException` or `Rule` red when I forgot to import them.  I don't think this is an Intellij configuration issue, because Intellij highlights similar errors in other projects on my machine.  

I'm wondering if something about how I forked and pulled down the repo is causing the errors to not show up.  I forked and pulled down the repo a second time, but still no luck.
Do either of you know of any issues that could be causing this problem?  (I am also googling furiously as we speak....)  Thanks.

**Comment by stkent** [comment]
When developing I usually edit the top-level `build.gradle` file to point to `src/test/java` rather than `build/gen/test/java`; that might help!
